packages/design/src/Form/components/PageSet/PageSet.tsx

```diff
@@ -23,7 +23,7 @@ const PageSet: PatternComponent<PageSetProps> = props => {
           })}
         />
       </nav>
-      <div className="tablet:grid-col-9">{props.children}</div>
+      <div className="tablet:grid-col-9 tablet:padding-left-4 padding-left-0 padding-bottom-3 padding-top-3 tablet:border-left tablet:border-base-lighter contentWrapper">{props.children}</div>
```

Comment:
I added "contentWrapper" when I was styling because it made it easy for me to find those components. Since I was using mainly utility classes to style the Edit Page, I found myself constantly having a tough time re-finding a component once I jumped to another. That class name isn't being used in any css rule at the moment.